### DISCUSSION TITLE
slide grid vertically

### DISCUSSION THREAD

**Discussion by shafschwd** [nolimits4web/swiper#5477] [Q&A]
is there anyway i can use grids vertically  [like here](https://swiperjs.com/demos/170-grid/core.html) but vertical
i used this value but its not working
`direction: "vertical",`

**Comment by Arhell** [comment]
You can make your own grid with html and css.
Put it all inside the item and it should work.

**Reply by shafschwd** [reply]
Hi thanks for replying
it didnt work for me if i put that `direction: "vertical",` on  my js whole site gets stuck
